---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[2], line 5
      3 import numpy as np
      4 from scipy.stats import cauchy
----> 5 from vec2d import Vec2d
      7 # Configuración del paseo aleatorio
      8 n_steps = 1000  # Número de pasos en la trayectoria

ModuleNotFoundError: No module named 'vec2d'
